packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -119,6 +128,59 @@ export function createTimeSeries(
   };
 }
 
+function transformResource(
+  resource: Resource,
+  projectId: string
+): { type: string; labels: { [key: string]: string } } {
```

Comment:
Consider using [MonitoredResource](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/ef3c4ad3eef08ec2c9e9f380e494248f13c73593/packages/opentelemetry-cloud-monitoring-exporter/src/types.ts#L68) as the return type here, getTypeAndMappings, and transformMetric to help readability.